tools/topology/topology2/include/common/pcm_caps.conf

```diff
@@ -88,8 +88,8 @@ Class.PCM."pcm_caps" {
 	periods_max		16
 	channels_min		2
 	channels_max		2
-	period_size_min		192
+	period_size_min		192 # Stereo, S16_LE, 48KHz, 1ms
 	period_size_max		16384
-	buffer_size_min		65536
-	buffer_size_max		65536
+	buffer_size_min		384 # period_size_min * 2
+	buffer_size_max		-1
```

Comment:
maybe use a more reasonable value? It's rather unlikely that the deep buffer will ever be more than 2s, that adds delays in user interaction to change tracks and apply volumes.

Infinity and beyond are not that useful for buffer sizes.